modules/nf-core/sratools/prefetch/environment.yml

```diff
@@ -5,3 +5,4 @@ channels:
   - defaults
 dependencies:
   - bioconda::sra-tools=3.0.8
+  - conda-forge::curl=8.5.0
```

Comment:
I believe we also need a matching container with these dependencies, no?

Reply:
Thanks for the useful feedback! curl is part of the container already. Do we need one nonetheless?